### DISCUSSION TITLE
PDFs

### DISCUSSION THREAD

**Discussion by BiasedGenesis** [furybee/chrome-tab-modifier#363] [Ideas]
Hello,

I installed Tab Modifier to be able to modify the names of PDFs I'm viewing in the chrome browser. The context menu button is there, but it doesn't do anything. I'm adding this as a feature request as it's probably more of an edge case for most users.

**Comment by sylouuu** [comment]
Hi @BiasedGenesis,

Already answered in #62.

Bests
